From 114c7abeec9b6fb88dd1ae23576730a4c2774ae2 Mon Sep 17 00:00:00 2001
From: Ronan Abhamon <ronan.abhamon@vates.fr>
Date: Wed, 16 Oct 2024 15:13:52 +0200
Subject: [PATCH] Prevent bad mypy error in TestMultiLUNISCSISR using
 formatted-string

Avoid:
```
error: Incompatible types in string interpolation (expression has type "object", placeholder has type "int | float | SupportsInt")  [str-format]
```

Signed-off-by: Ronan Abhamon <ronan.abhamon@vates.fr>
---
 tests/test_ISCSISR.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/test_ISCSISR.py b/tests/test_ISCSISR.py
index 39529be1..e71ac268 100644
--- a/tests/test_ISCSISR.py
+++ b/tests/test_ISCSISR.py
@@ -129,7 +129,7 @@ class TestMultiLUNISCSISR(unittest.TestCase):
             'tpgt': 'TPGT'
         }
         self.node_records = [(
-            "%s:%d" % (self.node2['ip'], self.node2['port']),
+            f"{self.node2['ip']}:{self.node2['port']}",
             self.node2['tpgt'],
             self.node2['iqn']
         )]
